src/main/java/com/google/cloud/tools/login/GoogleLoginState.java

```diff
@@ -58,6 +58,8 @@
 
   private static final String OAUTH2_NATIVE_CALLBACK_URL = GoogleOAuthConstants.OOB_REDIRECT_URI;
 
+  private static final String DEFAULT_APPLICATION_NAME = "google-ide-login";
```

Comment:
This is only used in one place to set another field, right? If so there's no real need for this constant.